.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/davidrodriguez/miniconda3/envs/q2dev/bin/python"
```

Comment:
Hey @David-Rod, even though you added this file to the gitignore, you will still have to manually remove it from this changeset. Let me know if you need a hand with that.